index.html

```diff
@@ -36,94 +51,102 @@ <h1>I’m Abdul <br> Glad to see you!</h1>
           <div class="content">
-            <h3>Multi-Post Stories</h3>
+            <h3 class="comp-title">Multi-Post Stories</h3>
             <div class="canopy">
               <h6>CANOPY</h6>
               <ul class="canopy-lists">
                 <li>Back End Dev</li>
                 <li>2015</li>
               </ul>
             </div>
+            <div class="project-desc">
             <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
             <ul class="categories">
-              <li>html</li>
-              <li>css</li>
-              <li>javaScript</li>
+              <li class="tag">html</li>
+              <li class="tag">css</li>
+              <li class="tag">javaScript</li>
             </ul>
-            <button>See Project</button>
+            <button class="btn" id="1">See Project</button>
           </div>
         </section>
 
         <section class="work-3">
           <div><img src="images/work3.png"></div>
           <div class="content">
-            <h3>Tonic</h3>
+            <h3 class="comp-title">Tonic</h3>
             <div class="canopy">
               <h6>CANOPY</h6>
               <ul class="canopy-lists">
                 <li>Back End Dev</li>
                 <li>2015</li>
               </ul>
             </div>
+            <div class="project-desc">
             <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
             <ul class="categories">
-              <li>html</li>
-              <li>css</li>
-              <li>javaScript</li>
+              <li class="tag">html</li>
+              <li class="tag">css</li>
+              <li class="tag">javaScript</li>
             </ul>
-            <button>See Project</button>   
+            <button class="btn" id="2">See Project</button>   
           </div>
           
         </section>
 
         <section class="work-4">
           <div><img src="images/work4.png"></div>
           <div class="content">
-            <h3>Multi-Post Stories</h3>
+            <h3 class="comp-title">Multi-Post Stories</h3>
             <div class="canopy">
               <h6>CANOPY</h6>
               <ul class="canopy-lists">
                 <li>Back End Dev</li>
                 <li>2015</li>
               </ul>
             </div>
+            <div class="project-desc">
             <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
             <ul class="categories">
-              <li>html</li>
-              <li>css</li>
-              <li>javaScript</li>
+              <li class="tag">html</li>
+              <li class="tag">css</li>
+              <li class="tag">javaScript</li>
             </ul>
-            <button>See Project</button>
+            <button class="btn" id="3">See Project</button>
           </div>
         </section>
```

Comment:
**[Optional]**

- [ ] Great work on implementing Popup dynamically via JavaScript 👏 It would be nice if you make all your project cards through javascript instead of making them a hardcoded item in your HTML file 👍 which you were supposed to do in this project 😊 The reason of doing this is that to make you learn how you can generate html using javascript because in future you will be working on API and all the data will come from APIs so in that case you will use the same technique which you were supposed to use here 🤔